Collivery/Setup/InstallData.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace MDS\Collivery\Setup;
+
+use Magento\Customer\Api\AddressMetadataInterface;
+use Magento\Eav\Model\Config;
+use Magento\Eav\Setup\EavSetup;
+use Magento\Framework\Setup\InstallDataInterface;
+use Magento\Framework\Setup\ModuleContextInterface;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+
+class InstallData implements InstallDataInterface
+{
+    /**
+     * Attribute Code of the Custom Attribute
+     */
+    const CUSTOM_ATTRIBUTE_CODE = 'custom';
+    /**
+     * @var EavSetup
+     */
+    private $eavSetup;
+    /**
+     * @var Config
+     */
+    private $eavConfig;
+    /**
+     * InstallData constructor.
+     * @param EavSetup $eavSetup
+     * @param Config $config
+     */
+    public function __construct(
+        EavSetup $eavSetup,
+        Config $config
+    ) {
+        $this->eavSetup = $eavSetup;
+        $this->eavConfig = $config;
+    }
+    public function install(ModuleDataSetupInterface $setup, ModuleContextInterface $context)
+    {
+        $installer = $setup;
```

Comment:
For the next developer arriving here (or you in 6 months), Please add comments for each step here and what it does